apps/wrapper/src/renderer/src/main.ts

```diff
@@ -2,7 +2,7 @@ import './assets/tailwind.css';
 import App from './App.svelte';
 
 const app = new App({
-  target: document.getElementById('app'),
+  target: document.body,
```

Comment:
Can we check if there is any other way to get back the `document.getElementById('app')` instead of using `document.body`? It threw us an error previous during production build but there should be another way to resolve this

Reply:
Would this work instead. Could not find anything online besides document.body

```
import App from './App.svelte';

const app = new App({
  target: document.querySelector('#app') as Element,
});
```